src/Interpreter.ts

```diff
@@ -6,43 +6,49 @@ import {
   getBallotStyle,
   getContests,
   getPrecinctById,
-  VotesDict,
 } from '@votingworks/ballot-encoder'
 import { strict as assert } from 'assert'
 import * as jsfeat from 'jsfeat'
 import { v4 as uuid } from 'uuid'
+import getVotesFromMarks from './getVotesFromMarks'
 import findContestOptions from './hmpb/findContestOptions'
 import findContests from './hmpb/findContests'
-import findTargets from './hmpb/findTargets'
-import { addVote } from './hmpb/votes'
+import findTargets, { TargetShape } from './hmpb/findTargets'
 import { detect } from './metadata'
 import {
+  BallotCandidateTargetMark,
+  BallotMark,
   BallotPageLayout,
   BallotPageMetadata,
+  BallotYesNoTargetMark,
   DetectQRCode,
-  InterpretedBallot,
-  InterpretedBallotCandidateTargetMark,
-  InterpretedBallotMark,
-  InterpretedBallotYesNoTargetMark,
-  Rect,
+  FindMarksResult,
+  Interpreted,
+  Point,
+  Size,
 } from './types'
-import { binarize } from './utils/binarize'
+import { binarize, PIXEL_BLACK, PIXEL_WHITE } from './utils/binarize'
+import crop from './utils/crop'
 import defined from './utils/defined'
 import { vh as flipVH } from './utils/flip'
 import { rectCorners } from './utils/geometry'
 import { map, reversed, zip, zipMin } from './utils/iterators'
-import diff, { ratio } from './utils/jsfeat/diff'
+import diff, { countPixels } from './utils/jsfeat/diff'
 import matToImageData from './utils/jsfeat/matToImageData'
 import readGrayscaleImage from './utils/jsfeat/readGrayscaleImage'
+import outline from './utils/outline'
 
 export interface Options {
   readonly election: Election
-  detectQRCode?: DetectQRCode
+  readonly detectQRCode?: DetectQRCode
+  readonly markScoreVoteThreshold?: number
 
   /** @deprecated */
   decodeQRCode?: DetectQRCode
 }
 
+export const DEFAULT_MARK_SCORE_VOTE_THRESHOLD = 0.2
```

Comment:
I kept the default value as 20% even though this PR makes it so scores will actually use the whole 0-1 range now. We can bump it up if needed, but in practice this default shouldn't really be used except in demos.